hikari/events/message_events.py

```diff
@@ -292,14 +356,27 @@ class GuildMessageCreateEvent(GuildMessageEvent, MessageCreateEvent):
     # <<inherited docstring from ShardEvent>>.
 
     message: messages.Message = attr.ib()
-
     # <<inherited docstring from MessageCreateEvent>>.
 
     @property
     def guild_id(self) -> snowflakes.Snowflake:
         # <<inherited docstring from GuildMessageEvent>>.
         # Always present in this event.
-        return typing.cast("snowflakes.Snowflake", self.message.guild_id)
+        guild_id = self.message.guild_id
+        assert isinstance(guild_id, snowflakes.Snowflake)
+        return guild_id
+
+    @property
+    def author(self) -> typing.Union[guilds.Member]:
```

Comment:
The `Union` here seems redundant, surely it should `guilds.Member` instead. Unless you mean `typing.Union[guilds.Member, users.User]`

Reply:
I think I meant the latter but now I think about it, that is redundant.

Reply:
```suggestion
    def author(self) -> guilds.Member:
```